manualdirectdebit.php

```diff
@@ -209,7 +209,7 @@ function manualdirectdebit_civicrm_custom($op, $groupID, $entityID, &$params) {
  *
  * @return bool
  */
-function isCustomGroupDirectDebit($groupID) {
+function isDirectDebitCustomGroup($groupID) {
```

Comment:
please prefix any non-hook function inside this file with **_manualdirectdebit_**, so in this case it would be **_manualdirectdebit_isDirectDebitCustomGroup**